model/simple/taskbehavior.go

```diff
@@ -171,12 +173,13 @@ func (tb *TaskBehavior) Done(ctx model.TaskContext) (notifyFlow bool, taskEntrie
 
 	linkInsts := ctx.GetToLinkInstances()
 
-	//Error branch already been handled, remove error branch from here
-	for i, link := range linkInsts {
-		if link.Link().Type() != definition.LtError {
-			linkInsts = append(linkInsts[:i], linkInsts[i+1:]...)
-		}
-	}
+	//todo review this, was ending flows prematurely on error
```

Comment:
It is required and it will not end the flow if there are more branches rather than just the error branch.   For the flow just end with an error branch it should be fine to end.  please add the below code. 
```
	for i, link := range linkInsts {
		if link.Link().Type() == definition.LtError {
			linkInsts = append(linkInsts[:i], linkInsts[i+1:]...)
		}
	}
```

Reply:
It fixed at last  commits, it looks good to me, please remove this code